### PR TITLE
ChromeOS needs KubeletInUserNamespace

### PR DESCRIPTION
**Pull request by dex4er**
I can run KIND cluster in ChromeOS sucesfully with this instruction. Actually it is not a specific to ChromeOS but it will be the same for running KIND in just any LXC container.

### CODE REVIEW

**Comment by stmcginnis** [comment]
/lgtm

I have not tested and do not have a way to easily do so, but it looks like everything necessary is added and if the submitter has tested and it works for them, that's good enough for me.

One caveat I'm thinking here - should we have a statement in the docs that this is an experimental or "unsupported" use case? Great that this works and is an option for folks, but I think it strays a little outside the [primary scope](https://kind.sigs.k8s.io/docs/contributing/project-scope/) of the project.

In other words, it might be good to make it clear that the maintainers won't spend a lot of time troubleshooting if a user files an issue having a problem running on ChromeOS.

**Review by BenTheElder — COMMENTED** [review]
/lgtm
/approve
Thank you!

**Comment by BenTheElder** [comment]
That's a good point @stmcginnis but we're also not currently requiring that for say, https://kind.sigs.k8s.io/docs/user/using-wsl2/ even though those docs are entirely user-contributed and not something we test (#1529). Docker Desktop on macOS is in a similar state currently as well of supported-but-not-actively-tested and only best-effort back and forth debugging via github.

**Comment by dex4er** [comment]
I see the difference between "unsupported" and "not working". I can thank you that you added the description of how to start KIND in LXC or ChromeOS even if I don't expect you can't troubleshoot such a setup.
